pkg/util/export/batch_processor.go

```diff
@@ -505,7 +506,7 @@ func (c *MOCollector) Stop(graceful bool) error {
 	c.stopOnce.Do(func() {
 		for len(c.awakeCollect) > 0 && graceful {
 			c.logger.Debug(fmt.Sprintf("doCollect left %d job", len(c.awakeCollect)))
-			time.Sleep(250 * time.Second)
+			time.Sleep(250 * time.Millisecond)
```

Comment:
Why make this shorter?

Reply:
here, waiting c.awakeCollect clean. Normally it will be consumed in short time (like 1 sec)